src/commands/config/common/default_values.rs

```diff
@@ -0,0 +1,18 @@
+use zksync_ethers_rs::types::H160;
+
+pub const DEFAULT_L1_RPC_URL: &str = "http://localhost:8545";
+pub const DEFAULT_L2_RPC_URL: &str = "http://localhost:3050";
+pub const DEFAULT_L2_EXPLORER_URL: &str = "http://localhost:3010";
+pub const DEFAULT_L1_EXPLORER_URL: &str = "";
+pub const DEFAULT_PRIVATE_KEY: &str =
+    "0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110";
```

Comment:
I think we should use another RICH wallet, the committer is the one that makes use of this `pk`, if we send any transaction with this wallet the local development chain would stop.

An available rich wallet:
```suggestion
    "0x850683b40d4a740aa6e745f889a6fdc8327be76e122f5aba645a5b02d0248db8";
```

And the address: `0xa13c10c0d5bd6f79041b9835c63f91de35a15883`

Also, we can use `Address::from_str("0xa13c10c0d5bd6f79041b9835c63f91de35a15883");` for simplicity.